content/neuvector-prime/azure/contents.lr

```diff
@@ -426,16 +426,19 @@ The AKS cluster already has the extension. Uninstall the extension as it suggest
    !["Uninstall extension"](./uninstall.png)
 
 
-### Checking the Kubernetes Service Extension version:
+###Upgrading a NeuVector Prime PAYG Cluster
 
-To see the most recent published extension versions, execute the following command in Cloud Shell.
+Updating the NeuVector version to the latest version supported by the marketplace listing can be accomplished using the "az k8s-extension update" command.
+
+The following az k8s-extension commands should be executed in a Cluster Cloud Shell:
```

Comment:
It does not have to be a Cluster Cloud Shell.  They could run this from anywhere that the az command is installed.

Reply:
should it be `az k8s-extension` ?

Reply:
The following az k8s-extension commands should be executed in a Cluster Cloud Shell or or Node which has access to Azure CLI: